diffrax/_solver/langevin_srk.py

```diff
@@ -0,0 +1,325 @@
+import abc
+from typing import Any, Callable, Generic, TypeVar
```

Comment:
Prefer `from collections.abc import Callable`. The `typing` one is now deprecated.

Reply:
Good to know, thanks!